src/GeniusYield/Unsafe.hs

```diff
@@ -0,0 +1,23 @@
+{-|
+Module      : GeniusYield.Unsafe
+Copyright   : (c) 2024 GYELD GMBH
+License     : Apache 2.0
+Maintainer  : support@geniusyield.co
+Stability   : develop
+
+-}
+module GeniusYield.Unsafe (
```

Comment:
This should really be under `GeniusYield.TxBuilder.IO.Unsafe`

Reply:
Have made this change in [this](https://github.com/geniusyield/atlas/pull/327/commits/89978b265c5650476d7db4c10d641a68bba12a42) commit!